### DISCUSSION TITLE
When will Mojo 🔥 0.6 be released?

### DISCUSSION THREAD

**Discussion by mojo-for-ai** [modular/modular#1359] [Q&A] [answered]
There are some major issues with Mojo 🔥 version 0.5. I wonder how long Mojo 🔥 will remain without updating.

**Comment by rarebreed** [comment]
Mojo is a baby.  I have seen you post some other Issues to be tracked for features that are reminiscent of rust.  Think about how long it took Rust to go from its initial open source release to 1.0. Rust's very first release was in 2010, and it had it's 1.0 release in 2015.  

Technically, mojo doesn't even have an open source release yet.  But if we count it as their first non-playground release, it's only 3 months old.  Also, Modular is smaller than Mozilla, so I have a feeling that they don't have as many engineers working on it as rust did while it was being worked on.

Keep in mind that Modular is having their ModCon convention very soon (today?).  They are going to be demoing some new features.  So there is a very good chance that they are holding back some new features to be able to demo them for the convention.

One of my big fears about mojo (and threads like these strengthen my fear) is that mojo was released way too early.  They should have waited until traits, lifetimes, references, and a more fleshed out standard library were released.  I am afraid that mojocians are going to get used to programming with raw pointers even though mojo is supposed to be a safe language.  And indeed, if mojo allows unsafe programming as easily as say C++, then I will stick to rust.

I do not mind that mojo is not open sourced yet.  I worked at Red Hat several years, and I know how hard it can be.  I also strongly believe in the BDFL model because again, I saw how some open source projects lacked a central guiding vision because it lacked a centra visionary and architect.

**Comment by soraros** [comment]
I share your excitement and enthusiasm for Mojo, and can't wait for 0.6. But I think it's coming with the ModCon in a few days.

That said, if you have some more concrete things (bugs, specific feature requests, improvement on the documents, etc.) in mind, please open separate issues for them, so we can discuss and maybe help the team to improve Mojo.

**Comment by scottamain** [comment, marked as answer]
We just released it today! 🔥
